src/main/java/com/google/api/generator/gapic/composer/SampleCodeHelperComposer.java

```diff
@@ -0,0 +1,149 @@
+                    "Note: Not Implement yet. Holder for lro Unary rpc method sample code.")))
+        .setIsSampleCode(true)
+        .build();
+  }
+
+  private static TryCatchStatement composePagedUnaryRpcMethodSampleCode(
+      Method method, List<MethodArgument> arguments, TypeNode clientType) {
+    // TODO(summerji): compose sample code for unary paged rpc method.
+    return TryCatchStatement.builder()
+        .setTryResourceExpr(assignClientVariableWithCreateMethodExpr(clientType))
+        .setTryBody(
+            Arrays.asList(
+                createLineCommentStatement(
+                    "Note: Not Implement yet. Holder for paged unary rpc method sample code.")))
+        .setIsSampleCode(true)
+        .build();
+  }
+
+  private static TryCatchStatement composeUnaryRpcDefaultMethodSampleCode(
+      Method method, TypeNode clientType) {
+    // TODO(summerji): compose sample code for unary default rpc method.
+    String content =
+        String.format(
+            "Note: Not Implement yet. Holder for unary %s rpc method sample code.",
+            (!method.hasLro() && !method.isPaged()
+                ? "default"
+                : (method.hasLro() ? "lro" : "paged")));
+    return TryCatchStatement.builder()
+        .setTryResourceExpr(assignClientVariableWithCreateMethodExpr(clientType))
+        .setTryBody(Arrays.asList(createLineCommentStatement(content)))
+        .setIsSampleCode(true)
+        .build();
+  }
+
+  // ==================================Helpers===================================================//
+
+  // Assign client variable expr with create client.
+  // e.g EchoClient echoClient = EchoClient.create()
+  private static AssignmentExpr assignClientVariableWithCreateMethodExpr(TypeNode clientType) {
+    return AssignmentExpr.builder()
+        .setVariableExpr(createVariableDeclExpr(getClientName(clientType), clientType))
+        .setValueExpr(
+            MethodInvocationExpr.builder()
+                .setStaticReferenceType(clientType)
+                .setReturnType(clientType)
+                .setMethodName("create")
+                .build())
+        .build();
+  }
+
+  private static String getClientName(TypeNode clientType) {
+    return JavaStyle.toLowerCamelCase(clientType.reference().name());
+  }
+
+  private static CommentStatement createLineCommentStatement(String content) {
+    return CommentStatement.withComment(LineComment.withComment(content));
+  }
+
+  private static VariableExpr createVariableExpr(String variableName, TypeNode type) {
+    return createVariableExpr(variableName, type, false);
+  }
+
+  private static VariableExpr createVariableDeclExpr(String variableName, TypeNode type) {
+    return createVariableExpr(variableName, type, true);
+  }
+
+  private static VariableExpr createVariableExpr(
+      String variableName, TypeNode type, boolean isDecl) {
+    return VariableExpr.builder()
+        .setVariable(createVariable(variableName, type))
```

Comment:
Why not just merge `createVariable` into here as well?

Reply:
I don't have preference here. Merged.